clinica/iotools/bids_utils.py

```diff
@@ -47,147 +47,152 @@ class StudyName(str, Enum):
 
 
 # -- Methods for the clinical data --
-# @ToDo:test this function
 def create_participants_df(
     study_name: StudyName,
     clinical_spec_path,
     clinical_data_dir,
     bids_ids,
     delete_non_bids_info=True,
-):
-    """Create the file participants.tsv.
+) -> DataFrame:
+    """Create the participants dataframe that will be saved as tsv.
 
     Args:
         study_name: name of the study (Ex. ADNI)
-        clinical_spec_path: path to the clinical file
+        clinical_spec_path: path to the clinical file for study specifications
         clinical_data_dir: path to the directory where the clinical data are stored
         bids_ids: list of bids ids
         delete_non_bids_info: if True delete all the rows of the subjects that
         are not available in the BIDS dataset
 
     Returns: a pandas dataframe that contains the participants data
     """
-    import os
     from os import path
 
     import numpy as np
     import pandas as pd
 
     from clinica.iotools.converters.adni_to_bids.adni_utils import load_clinical_csv
     from clinica.utils.stream import cprint
+    # todo : diff between what is in ADNI raw and BIDS can do get subjects (? not same naming type)
+    # todo : used in all converters ? (only OASIS and ADNI, what happens if used elsewhere?)
+    # todo : typing, test
 
-    fields_bids = ["participant_id"]
-    prev_location = ""
-    index_to_drop = []
-    subjects_to_drop = []
     study_name = StudyName(study_name)
     location_name = f"{study_name.value} location"
 
-    # Load the data from the clincal specification file
+    # Load the data from the clinical specification file
```

Comment:
I don't know if this changes something or not, but I made some changes related to those files in #1140 
Basically, I moved these files to the converter they apply to and used consistent names instead of encoding the converter within the name.